### ISSUE TITLE
Support / Activity?

### ISSUE THREAD

**Issue by gklesczewski** [dotproject/dotProject#200]
Hi Everyone, I was looking to see who is working on the dotProject codebase these days?

I am looking to set up some automated testing so I can help with the PHP version upgrades to get the platform up to date.

**Comment by phjbouwhuis** [comment]
I do not believe this project is very active. I've been using it for the past 8 years, and it still works fine for me. There have been a few security updates, and you need to keep a lookout on the PHP version, but other than that it's a great system.
